### PR TITLE
add left and right force launch

### PR DESCRIPTION
**Pull request by Kanazawanaoaki**
I add left and right version of virtual_force_publisher.launch

### CODE REVIEW

**Comment by knorth55** [comment]
can you add them in `jsk_robot/jsk_pr2_startup/pr2_bringup.launch` to run this launch every time?

**Comment by Kanazawanaoaki** [comment]
@knorth55 
For the time being, I tried to put Pull req on jsk_robot.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1251

**Comment by Kanazawanaoaki** [comment]
The following data can be obtained by applying the kitchen knife to the cutting board and carrot.  
gopro video : https://youtu.be/NIpRjineEps

|force of y direction|force of z direction|
|---|---|
|wrench_y_graph_knife_wrench_test_20200925|wrench_z_graph_knife_wrench_test_20200925|

**Comment by knorth55** [comment]
this PR may break backward compatibility.
You should keep `virtual_force_publisher.launch` as it is, and include it in `[left|right]_virtual_force_publisher.launch` and include the left and right ones into `dualarm_force_publisher.launch`.

```
virtual_force_publisher -> left_virtual_force_publisher.launch -> dualarm_virtual_force_publisher.launch 
                         |-> right_virtual_force_publisher.launch -|
```

**Comment by Kanazawanaoaki** [comment]
I didn't think about backward compatibility, so I fixed it.  
https://github.com/jsk-ros-pkg/jsk_robot/pull/1251 also includes the newly created `dualarm_virtual_force_publisher.launch`.

File: virtual_force_publisher/launch/left_virtual_force_publisher.launch [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <group ns="$(arg namespace)">
+    <node pkg="virtual_force_publisher" type="virtual_force_publisher" name="virtual_force_publisher">
+      <remap from="joint_states" to="/$(arg input)"/>
+      <param name="publish_frequency" value="$(arg publish_frequency)"/>
+      <param name="time_constant" value="$(arg time_constant)"/>
+      <param name="root" value="$(arg root)"/>
+      <param name="tip" value="$(arg tip)"/>
+    </node>
+  </group>
+</launch>
```
Comment (knorth55) [inline virtual_force_publisher/launch/left_virtual_force_publisher.launch:17]: how about this?

```suggestion
<launch>
  <arg name="namespace" default="left_endeffector" />
  <arg name="publish_frequency" default="50.0"/>
  <arg name="time_constant" default="0.3"/>
  <arg name="root" default="torso_lift_link"/>
  <arg name="tip"  default="l_gripper_tool_frame"/>
  <arg name="input" default="joint_states"/>
  <include file="$(find virtual_force_publisher)/launch/virtual_force_publisher.launch" >
    <arg name="namespace" value="$(arg namespace)" />
    <arg name="publish_frequency" value="$(arg publish_frequency)"/>
    <arg name="time_constant" value="$(arg time_constant)"/>
    <arg name="root" value="$(arg root)"/>
    <arg name="tip"  value="$(arg tip)"/>
    <arg name="input" value="$(arg input)"/>
  </include>
</launch>
```

File: virtual_force_publisher/launch/right_virtual_force_publisher.launch [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <group ns="$(arg namespace)">
+    <node pkg="virtual_force_publisher" type="virtual_force_publisher" name="virtual_force_publisher">
+      <remap from="joint_states" to="/$(arg input)"/>
+      <param name="publish_frequency" value="$(arg publish_frequency)"/>
+      <param name="time_constant" value="$(arg time_constant)"/>
+      <param name="root" value="$(arg root)"/>
+      <param name="tip" value="$(arg tip)"/>
+    </node>
+  </group>
+</launch>
```
Comment (knorth55) [inline virtual_force_publisher/launch/right_virtual_force_publisher.launch:17]: how about this?

```suggestion
<launch>
  <arg name="namespace" default="right_endeffector" />
  <arg name="publish_frequency" default="50.0"/>
  <arg name="time_constant" default="0.3"/>
  <arg name="root" default="torso_lift_link"/>
  <arg name="tip"  default="r_gripper_tool_frame"/>
  <arg name="input" default="joint_states"/>
  <include file="$(find virtual_force_publisher)/launch/virtual_force_publisher.launch" >
    <arg name="namespace" value="$(arg namespace)" />
    <arg name="publish_frequency" value="$(arg publish_frequency)"/>
    <arg name="time_constant" value="$(arg time_constant)"/>
    <arg name="root" value="$(arg root)"/>
    <arg name="tip"  value="$(arg tip)"/>
    <arg name="input" value="$(arg input)"/>
  </include>
</launch>
```

**Review by knorth55 — APPROVED** [review]

**Pull request merged**